friends/src/FriendsList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class FriendsList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then(response => {
+        this.setState(() => ({ friends: response.data }));
+        console.log(this.state.friends);
+      })
+      .catch(error => {
+        console.error("Server Error", error);
+      });
+  }
+
+  render() {
+    return (
+      <div className="friendTable">
+        {this.state.friends.map(friend => (
+          <FriendDetails friend={friend} />
```

Comment:
Remember to add your `key` prop when mapping. You could do something like `<FriendDetails friend={friend} key={friend.id} />`